skycultures/arabic/dso_names.fab

```diff
@@ -0,0 +1,13 @@
+#NGC 869 |_("المعصم 1")
+#NGC 884 |_("المعصم 2") 
+#M 45    |_("الثريا") 
+#M 44    |_("النثرة")
+#M 7     |_("إبرة العقرب")
+NGC 869 |_("The Wrist 1")
+NGC 884 |_("The Wrist 2")
+NGC 869 |_("The She-Camel's Brand 1")
+NGC 884 |_("The She-Camel's Brand 2")
+# TRANSLATORS: Al-Thurayya is a proper name
+M 45    |_("Al-Thurayya")
+M 44    |_("The Lion's Nostrils")
+M 7     |_("The Scorpion'S Stinger")
```

Comment:
--> The Scorpion's Sting ?

Reply:
Fixed capital S.
As I checked, sting is the injury, and stinger is tip of the scorpion's tail.

Reply:
"Sting" can be both and is I think more common. But you can leave it.

Reply:
Changed to "Sting">